releases/release-6.2.0.md

```diff
@@ -0,0 +1,450 @@
+---
+title: TiDB 6.2.0 Release Notes
+---
+
+# TiDB v6.2.0 Release Notes
+
+Release date: August 16, 2022
+
+TiDB version: 6.2.0
+
+In 6.2.0, the key new features or improvements are as follows:
+
+* TiDB Dashboard supports visual execution plans, allowing more intuitive display of execution plans.
+* TiDB Dashboard adds a Monitoring page for more efficient performance analysis and tuning.
+* TiDB supports lock view to show the waiting information of optimistic transactions, facilitating quick locating of lock conflicts.
+* TiFlash supports a newer version of storage format, enhancing stability and performance.
+* The Fine Grained Shuffle feature allows parallel execution of window functions in multiple threads.
+* A new concurrent DDL framework: Less DDLs blocked and higher execution efficiency.
+* TiKV supports automatically tuning the CPU usage, thus ensuring stable and efficient database running.
+* Point-in-Time Recovery (PITR) is introduced to restore a snapshot of a TiDB cluster to a new cluster from any given time point in the past.
+* TiDB Lightning supports importing data to productions clusters in physical import mode.
```

Comment:
1.”TiDB Lightning supports importing data to productions clusters in physical import mode.“  needs to be modified to "TiDB Lightning supports importing data to productions clusters in table-level physical import mode."

2.Add a new feature of TiDB MD
When DM migrates incremental  data from MySQL to TiDB, it supports incremental data verification to check data consistency.

Reply:
We have already mentioned continuous data verification in this file, see https://github.com/pingcap/docs/pull/9906/files#diff-d29bf2d43d9f4861e58933e6deea892bc12416c947be01e43ed742443c64f396R180